### ISSUE TITLE
[FR] Errors should appear on both serial ports.

### ISSUE THREAD

**Issue by wormley** [MarlinFirmware/Marlin#18200]
### Description

I believe in the past when an error occurred like 'Printer halted' it appeared on both serial ports, this way a TFT or other smart display on SERIAL_PORT_2 could notify the user in addition to whatever was connected to the main serial port. It looks like this feature was removed at some point in the past. 

Ideally there might also be a configuration option to force boot messages to go to both as well.(And maybe also just general SERIAL_ECHOs) although this is more for those of us developing who are too lazy to swap our ports so we have not-USB on the main port and can capture boot messages.

### Feature Workflow

The messages when Marlin stops:

Error:Heating failed, system stopped! Heater_ID: bed
Error:Printer halted. kill() called!

Should appear on both ports, currently the device on the second port has no idea that that something bad happened other than the system is no longer accepting commands.

**Comment by ellensp** [comment]
I disagree with this strongly. You don't know what is on serial port 2. This could make things worse confusing or even crashing what is on the serial port 2.
SERIAL_PORT is always the main serial port that you want error messages to go to.
SERIAL_PORT_2 is a secondary serial port for other things
If you control things via a TFT you should make sure it is talking to SERIAL_PORT
If you control things via USB, you should make sure that is the SERIAL_PORT
